adapters/handlers/rest/handlers_schema.go

```diff
@@ -250,8 +252,10 @@ func (s *schemaHandlers) restoreSnapshot(params schema.SchemaObjectsSnapshotsRes
 func (s *schemaHandlers) restoreSnapshotStatus(params schema.SchemaObjectsSnapshotsRestoreStatusParams,
 	principal *models.Principal,
 ) middleware.Responder {
-	// TODO implement
-	return nil
+	status := s.manager.RestoreStatus
+	err := s.manager.RestoreError
+	txt := fmt.Sprintf("%v - %v", status, err)
+	return schema.NewSchemaObjectsSnapshotsRestoreStatusOK().WithPayload(&models.SnapshotRestoreMeta{Status: &txt, ClassName: params.ClassName, Error: err.Error(), ID: params.ID, Path: "FIXME must set path", StorageName: params.StorageName})
```

Comment:
can format a little bit this line, so that this line would not be so long to read? at least break down `&models.SnapshotRestoreMeta` into separate lines, so that it would be easy to read